src/game-views/OenologyGame.css

```diff
@@ -1,16 +1,29 @@
 .OenologyGame {
     display: grid;
-    grid-template-columns: 1fr 530px;
-    grid-template-rows: 1fr 320px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 70vh 280px 1fr;
+    grid-template-rows: calc(100vh - 36px - 280px) 280px 1fr;
     grid-template-areas:
-        "main sidebar"
-        "bottom sidebar";
+        "main"
+        "bottom"
+        "sidebar";
     flex: 1 1 auto;
     min-height: 1px;
 }
 
+@media (min-width: 950px) {
+    .OenologyGame {
+        grid-template-columns: 1fr 510px;
```

Comment:
how come this was changed from 510 (and 900 below)? did you check if those work in safari and firefox, especially with fat scrollbars?